addons/mail/static/src/models/thread/thread.js

```diff
@@ -1280,6 +1288,19 @@ function factory(dependencies) {
             ],
         }),
         id: attr(),
+        /**
+         * Determines whether this thread is a `mail.channel` qualified as chat.
```

Comment:
`States` not `Determines`. Changing the value has to effect to change the type of channel to chat.